core/jvm/src/Instant.kt

```diff
@@ -43,8 +44,8 @@ public actual class Instant internal constructor(internal val value: jtInstant)
     public actual operator fun minus(duration: Duration): Instant = plus(-duration)
 
     public actual operator fun minus(other: Instant): Duration =
-            Duration.seconds(this.value.epochSecond - other.value.epochSecond) + // won't overflow given the instant bounds
-            Duration.nanoseconds(this.value.nano - other.value.nano)
+        (this.value.epochSecond - other.value.epochSecond).seconds + // won't overflow given the instant bounds
```

Comment:
Minor: I'd probably use in such cases a more verbose `.toDuration(DurationUnit.SECONDS)` to sound less tautological.

Reply:
Tried this and didn't like the result, looked to me as much more cumbersome than both `Duration.seconds(...)` and `.seconds`, so the intent is obscured. `.seconds` is not pretty, but very obvious.